atom/browser/web_contents_preferences.h

```diff
@@ -29,6 +29,11 @@ namespace atom {
 class WebContentsPreferences
     : public content::WebContentsUserData<WebContentsPreferences> {
  public:
+  enum DeprecationStatus {
```

Comment:
Seems like `Deprecation` is redundant here, `Status` would be sufficient and terser. E.g. we could add `Experimental` to the enum at some point

Also this probably should be an `enum class` rather than an `enum`

Reply:
+1 for the enum class